src/Core/UUID.h

```diff
@@ -11,6 +11,40 @@ namespace UUIDHelpers
     /// Generate random UUID.
     UUID generateV4();
 
+    const size_t HighBytes =
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+        0;
+#else
+        1;
+#endif
+
+    const size_t LowBytes =
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
+        1;
+#else
+        0;
+#endif
+
+    inline uint64_t getUUIDHigh(const UUID & uuid)
```

Comment:
(we are in namespace `UUIDHelpers` already, could name this method `getHighBytes()`)